docs/sending-techniques/sending-data-cli-runner/sending-data-cli-runner.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_label: Overview
+---
+
+# Run Slack CLI commands in GitHub Actions workflows 
+
+This technique uses the Slack CLI in GitHub Actions to run commands through workflows.
+
+Setting up a CI/CD pipeline [requires](https://docs.slack.dev/tools/slack-cli/guides/authorizing-the-slack-cli/#ci-cd) authorization using a service token. [Service tokens](https://docs.slack.dev/tools/slack-cli/guides/authorizing-the-slack-cli/#obtain-token) are long-lived, non-rotatable user tokens that don’t expire.
+
+## Setup
+
+1. Add your service token as a [repository secret](https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-secrets#creating-secrets-for-a-repository) called SLACK_SERVICE_TOKEN.
+2. Add this workflow to your repository. GitHub Actions workflow files must be stored in the .github/workflows directory
+
+```yaml
+name: Slack CLI Command Runner
+on:
+  workflow_dispatch:
+    inputs:
+      command:
+        description: 'Slack CLI command to run'
+        type: string
+        default: ""
+        required: true
+      verbose:
+        description: 'Verbose flag'
+        type: boolean
+        default: false
+        required: false
+      cli_version:
+        description: 'CLI version'
+        type: string
+        default: "latest"
+        required: false
+      app_id: 
+        description: "App ID"
+        type: string
+        default: ""
+        required: false
+
+jobs:
+  run-slack-cli:
+    uses: slackapi/slack-github-action/.github/workflows/cli-runner.yml@main
+    with:
+      command: ${{ github.event.inputs.command }}
+      verbose: ${{ github.event.inputs.verbose == 'true' || github.run_attempt > 1 }}
+      cli_version: ${{ github.event.inputs.cli_version }}
+      app_id: ${{ github.event.inputs.app_id }}
+    secrets: inherit
```

Comment:
```suggestion
```

🪓 thought: We shouldn't recommend inherited secrets since this can cause confusion behaviors of which inputs are being used and for what. It might also let folks change what value a token is saved as-